src/main/java/com/esri/samples/featurelayers/feature_collection_layer_query/README.md

```diff
@@ -0,0 +1,38 @@
+<h1>Feature Collection Layer Query</h1>
+
+<p>Create a feature collection layer which shows the result of a SQL query from a service feature table.</p>
+
+<p><img src="FeatureCollectionLayerQuery.png"/></p>
```

Comment:
That's some very exciting data right there lol

Reply:
I know right haha